Jenkins build failures - Unnecessary "else" after "raise"

https://github.com/holland-backup/holland/blob/master/holland/core/util/lock.py#L34
https://jenkins.hollandbackup.org/job/holland/job/master/20/execution/node/40/log/
> holland/core/util/lock.py:28:8: R1720: Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it (no-else-raise)
> holland/core/util/lock.py:45:8: R1720: Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it (no-else-raise)


https://github.com/holland-backup/holland/blob/master/contrib/holland-commvault/holland_commvault/commvault.py#L159
https://jenkins.hollandbackup.org/job/holland/job/master/20/execution/node/41/log/
> contrib/holland-commvault/holland_commvault/commvault.py:153:8: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)

--- a/.pylintrc
+++ b/.pylintrc
@@ -215,8 +215,8 @@ min-public-methods=1
 [EXCEPTIONS]
 
 # Exceptions that will emit a warning when caught.
-overgeneral-exceptions=BaseException,
-                       Exception
+overgeneral-exceptions=builtins.BaseException,
+                       builtins.Exception
 
 
 [FORMAT]

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -12,7 +12,7 @@ pipeline {
         sh '''apt-get update
 apt-get install -y python3-pip python3-psycopg2 rsync
 pip3 install configobj
-pip3 install pylint
+pip3 install 'pylint>=2.17.0,<3.0.0'
 pip3 install six
 python3 --version'''
       }

--- a/contrib/holland-commvault/holland_commvault/commvault.py
+++ b/contrib/holland-commvault/holland_commvault/commvault.py
@@ -156,8 +156,7 @@ def main():
         except IOError:
             logging.info("Holland (commvault agent) failed to open/read status file")
             return 1
-        else:
-            status.close()
+        status.close()
         return ret
     except IOError as ex:
         logging.warning("Holland (commvault agent) IOError: %r", ex)

--- a/holland/core/util/lock.py
+++ b/holland/core/util/lock.py
@@ -31,8 +31,7 @@ class Lock(object):
         except IOError as exc:
             self.lock = None
             raise LockError(str(exc), exc)
-        else:
-            return True
+        return True
 
     def is_locked(self):
         """Check for lock"""
@@ -48,5 +47,4 @@ class Lock(object):
             self.lock = None
         except IOError as exc:
             raise LockError(str(exc), exc)
-        else:
-            return True
+        return True

--- a/plugins/holland.lib.mysql/holland/lib/mysql/option/base.py
+++ b/plugins/holland.lib.mysql/holland/lib/mysql/option/base.py
@@ -2,6 +2,7 @@
 
 http://dev.mysql.com/doc/refman/5.1/en/option-files.html
 """
+
 import codecs
 import logging
 import os
@@ -13,7 +14,7 @@ LOG = logging.getLogger(__name__)
 
 def merge_options(*defaults_files):
     """Merge multiple defaults files together"""
-    defaults_config = dict(client={})
+    defaults_config = {"client": {}}
 
     def merge(dst_dict, src_dict):
         """Merge two dictionaries non-destructively"""

